### DISCUSSION TITLE
Stock Location and Stock Location Types

### DISCUSSION THREAD

**Discussion by LarryJoy** [inventree/InvenTree#8200] [Q&A]
This is what I have set up for admin > Stock > Stock Location Types:
![image](https://github.com/user-attachments/assets/75324917-bd48-4f99-ba00-3f44bf52d972)
And this is what I get for choices of Location Types:
![image](https://github.com/user-attachments/assets/44a2a55a-4f7d-4288-b56c-d178e53d7917)
Again, there seems to be a disconnect between what is in the admin Stock setup and what is available when trying to make edits to the stock part. How come? What is going on? What do I need to do to make this work properly?

**Comment by SergeoLacruz** [comment]
I think you are mixing up stock location type with stock location. The stock location, and even the default location, is something like room04 shelf04 box07. The stock location type is the physical medium, like plastic box, card box, reel, bucket, ...

**Comment by LarryJoy** [comment]
After mucking around with the InvenTree program, after getting it downloaded and installed properly, I did figure out the terminology and get things to work.--Thanks for your eventual reply.
